Assets/MixedRealityToolkit/Utilities/Editor/EditorProjectUtilities.cs

```diff
@@ -18,6 +18,7 @@ public static class EditorProjectUtilities
         static EditorProjectUtilities()
         {
             CheckMinimumEditorVersion();
+            PackageManifestUpdater.EnsureMSBuildForUnity();
```

Comment:
Should we make this part of the configurator instead?

Reply:
not sure. what is the consensus from the reviewers?

Reply:
My thought is making this opt-in. We can gate everything around the presence of the adapter, with remoting / additional feature support (like autoplanar) gated behind an opt-in.

Reply:
tracking this with #6723